packages/console/src/pages/SignInExperience/index.tsx

```diff
@@ -54,6 +56,18 @@ const SignInExperience = () => {
     toast.success(t('application_details.save_success'));
   });
 
+  if (!settings && !settingsError) {
+    return <div>loading</div>;
+  }
+
+  if (settingsError) {
+    return <div>{settingsError.body.message}</div>;
```

Comment:
Will it be handled by the default error page?

Reply:
Not sure about that, align them later.